docs/docs/exporting.md

```diff
@@ -69,3 +71,30 @@ loki-distributor-deployment
 etc.
 
 You can also use a different file extension by providing `--extension='yml'`, for example.
+
+
+## Multiple environments
+
+Tanka can also export of multiple inline environments, as showcased in [Use case: consistent inline
+environments](https://tanka.dev/inline-environments#use-case-consistent-inline-environments). This follows the same
```

Comment:
```suggestion
environments](/inline-environments#use-case-consistent-inline-environments). This follows the same
```

Otherwise `react-router` doesn't handle this link. Also deploy previous would be inconsistent